internal/pathutil/pathutil.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package pathutil provides utility functions around file system paths.
+package pathutil
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+// Expand expands the path.
+func Expand(path string) (string, error) {
+	path, err := homedir.Expand(path)
+	if err != nil {
+		return "", err
+	}
+	return filepath.Abs(os.ExpandEnv(path))
```

Comment:
No need to abs

Reply:
Why not abs?

Reply:
Relative paths work fine now. We needed the abs stuff when using bazel because relative from the bazel running dir would not be valid

Reply:
Done.